rabbit-core/src/main/java/org/ohdsi/rabbitInAHat/dataModel/Field.java

```diff
@@ -114,11 +114,11 @@ public String getComment() {
 		return comment;
 	}
 
-	public int getMaxLength() {
+	public Integer getMaxLength() {

```

Comment:
Is this the maximum length of a String or the maximum number of rows? If it is the latter, it may be wise to replace this by a Long.

Reply:
Never mind, I see it's string length.